### PR TITLE
readme: fix typos

### PR DESCRIPTION
**Pull request by spageektti**
Hi. This PR fixes some typos in the `readme.md` file.

### CODE REVIEW

**Comment by Saithe6** [comment]
Some of these are likely not typos. "Look around with mouse" is a place where it is perfectly reasonable to drop the articles

File: readme.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ A 3D Linux desktop environment (which can also be a game engine)
 ## Usage
 
 ### Navigate the 3d space
-Look around with mouse
+Look around with a mouse
```
Comment (collinalexbell) [inline readme.md:14]: Replace w "the".

**Review by collinalexbell — APPROVED** [review]

**Comment by collinalexbell** [comment]
Please make the one fix I requested. Pre-approved.

**Comment by collinalexbell** [comment]
❤️. I merged and changed "a" to "the" in front of "mouse" in a subsequent commit.

**Pull request merged**